terraform/rock.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# -----------------------------------------------------------------------------
+# CodeDeploy
+
+systemctl start codedeploy-agent
+
+# -----------------------------------------------------------------------------
+# JouralD -> CloudWatch
+
+cat <<EOF > /etc/systemd-cloud-watch.conf
+log_group = "rock"
+EOF
+
+# Looks like there's a but that results in a lot of log output from this library.
+#
+# Fix is in this PR: https://github.com/advantageous/systemd-cloud-watch/pull/16
+# systemctl start systemd-cloud-watch
```

Comment:
This is the user data file that will be executed inside the EC2 instance when you provision with Terraform.

Starting services, adding configuration files goes here.